src/librustc/traits/object_safety.rs

```diff
@@ -47,13 +47,15 @@ impl ObjectSafetyViolation {
                 "the trait cannot use `Self` as a type parameter \
                  in the supertraits or where-clauses".into(),
             ObjectSafetyViolation::Method(name, MethodViolationCode::StaticMethod) =>
-                format!("method `{}` has no receiver", name).into(),
-            ObjectSafetyViolation::Method(name, MethodViolationCode::ReferencesSelf) =>
-                format!("method `{}` references the `Self` type \
-                         in its arguments or return type", name).into(),
-            ObjectSafetyViolation::Method(name,
-                                            MethodViolationCode::WhereClauseReferencesSelf(_)) =>
-                format!("method `{}` references the `Self` type in where clauses", name).into(),
+                format!("associated function `{}` has no `self` parameter", name).into(),
+            ObjectSafetyViolation::Method(name, MethodViolationCode::ReferencesSelf) => format!(
+                "method `{}` references the `Self` type in its arguments or return type",
```

Comment:
This change never occurs in the `--bless` and I suspect "arguments" should be "parameters".

Reply:
[It did](https://github.com/rust-lang/rust/pull/64110/files#diff-acc2dab55a045afa40b3c9e94c09b1a9)? (It should be `parameters,` making the change.)

Reply:
Not in the first commit (which is what I wrote the comment wrt.) =P

Reply:
:)